src/ensembl/src/shared/components/communication-framework/contact-us/contact-us-form/initial/ContactUsInitialForm.tsx

```diff
@@ -106,23 +116,36 @@ const reducer = (state: State, action: Action): State => {
       const newFiles = [...state.files];
       newFiles.splice(action.payload, 1);
       return { ...state, files: newFiles };
+    case 'replace-state':
+      return action.payload;
     default:
       return state;
   }
 };
 
+const FORM_NAME = 'contact-us-general';
+
 const ContactUsInitialForm = () => {
   const [state, dispatch] = useReducer(reducer, initialState);
   const [isChallengeCompleted, setIsChallengeCompleted] = useState(false);
   const [emailFieldValid, setEmailFieldValid] = useState(true);
   const [emailFieldFocussed, setEmailFieldFocussed] = useState(false);
-  const [isSubmitted, setIsSubmitted] = useState(false);
+  const [submissionState, setSubmissionState] = useState<LoadingState>(
+    LoadingState.NOT_REQUESTED
+  );
 
   const formRef = useRef<HTMLFormElement>(null);
   const emailFieldRef = useRef<HTMLInputElement | null>(null);
   const stateRef = useRef<typeof state>();
   stateRef.current = state;
 
+  const { clearSavedForm } = useSavedForm({
+    formName: 'initial-form',
```

Comment:
Here and on line 126, why are we using two different form names ?

Reply:
My forgetfulness :smiley: I will use the one assigned to the constant.

Reply:
✅